keps/sig-node/127-user-namespaces/README.md

```diff
@@ -250,6 +312,56 @@ limit the number of pods using user namespaces to `min(maxPods, 1024)`. This
 leaves us plenty of host UID space free and this limits is probably never hit in
 practice. See UNRESOLVED for more some UNRESOLVED info we still have on this.
 
+##### pkg/volume changes for phase I
+
+We need the files created by the kubelet for these volume types to be readable
+by the user the pod is mapped to in the host. Luckily, Kuberentes already
+supports specifying an [fsUser and fsGroup for volumes][volume-mounter-args] so
+the changes needed here are rather small.
```

Comment:
>> From the comment there:
>>
>> // Currently only supported with projected service account tokens.
>> Isn't that a problem? Doesn't that mean it will work only for projected SA tokens?

> No, we will change that as we explain in this section. In particular, see here, the second thing we need to do here is to make these volume types honor this setting (today only SA tokens honor this): https://github.com/kubernetes/enhancements/pull/3275/files/db6ca5fc202e1d7af37ed339232f7395a61d67c8#diff-3327dfddda18ef4717e48d017e35c7badbd20e1c59d3b02f028bd9d520d8351aR350-R351

Let me rephrase my question.
I agree it seems doable (and maybe the changes are indeed small), but we seem to currently support that only for projected SA tokens.

Can you rephrease from "k8s already supports..." to something like
"k8s already supports ... *for projected SA tokens* so we just need to extend that to other volume types"
[or sth like that]?

Reply:
Ohh, I see your point now. Changing it to that :)